app/code/Magento/Customer/Controller/Account/ForgotPasswordPost.php

```diff
@@ -79,6 +88,7 @@ public function execute()
                     AccountManagement::EMAIL_RESET
                 );
             } catch (NoSuchEntityException $exception) {
+                $this->logger->info($exception);
```

Comment:
Could you explain why do you need to log this exception and wasn't needed before?